playbooks/utils/vSphere_audit.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Get info from vSphere
+  hosts: localhost
+
+  vars_files:
+    - ../../group_vars/vsphere/vault.yml
+    - ../../group_vars/vsphere/{{ runtime_env | default('staging') }}.yml
+
+  tasks:
+    - name: Gather info on templates
+      community.vmware.vmware_vm_info:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        validate_certs: false
+        show_datacenter: true
+        show_cluster: true
+        show_esxi_hostname: true
+        vm_type: template
+      register: template_info
+
+    - name: View template info
+      ansible.builtin.debug:
+        var: template_info
+
+    - name: Gather info on hosts
```

Comment:
can we distinguish between ESXi host and host. I think we are asking for the ESXi host but not sure.

Reply:
Good idea, thanks. Addressed in c1e12912b6ff1ba3f55d0bfb0f37a5e98ce8f3b5.